.eslintrc

```diff
@@ -1,5 +1,21 @@
 {
   "extends": "airbnb",
+  "ecmaFeatures": {
+    "jsx": true
+  },
+
+ "globals": {
+      "React": true
```

Comment:
indentation; but also, React shouldn't be a global - every file should explicitly require it.